.lycheeignore

```diff
@@ -1 +1,2 @@
 https://localhost:9200*
+http://localhost:*
```

Comment:
Let's consolidate this into a single `https://localhost:*` and change the prometheus URL below to an `https://` one so it catches both?